### PR TITLE
Creating koku-metrics-operator namespace and installing koku-metrics operator

### PR DESCRIPTION
**Pull request by skanthed**

### CODE REVIEW

File: cluster-scope/base/operators.coreos.com/subscriptions/koku-metrics-operator/subscription.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: koku-metrics-operator-sub
```
Comment (naved001) [inline cluster-scope/base/operators.coreos.com/subscriptions/koku-metrics-operator/subscription.yaml:4]: you should just name it `koku-metrics-operator`

File: cluster-scope/base/core/namespaces/koku-metrics-operator/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: koku-metrics-operator
+resources:
+- namespace.yaml
```
Comment (naved001) [inline cluster-scope/base/core/namespaces/koku-metrics-operator/kustomization.yaml:5]: needs a new line at the end of the file. 

also, a few other files in this PR need newline.

**Review by naved001 — CHANGES_REQUESTED** [review]

File: cluster-scope/base/operators.coreos.com/subscriptions/koku-metrics-operator/subscription.yaml (hunk 1 above)
Comment (skanthed) [reply cluster-scope/base/operators.coreos.com/subscriptions/koku-metrics-operator/subscription.yaml]: Updated the name.

**Comment by skanthed** [comment]
Creating new PR as some unexpected changes were being pushed that were made to the pre-commit config file.

**Pull request closed without merging**